contrib/network/connection_balance/dlb/source/connection_balancer_impl.h

```diff
@@ -92,11 +92,17 @@ class DlbConnectionBalanceFactory : public Envoy::Network::ConnectionBalanceFact
 
   std::string name() const override { return "envoy.network.connection_balance.dlb"; }
 
+  // Log error info in warn level and fallback based on fallback policy.
+  Envoy::Network::ConnectionBalancerSharedPtr fallback(const std::string& message);
+
   // Init those only when Envoy start.
   int domain_id, ldb_pool_id, dir_pool_id, tx_queue_id;
+  // The default value is None.
+  envoy::extensions::network::connection_balance::dlb::v3alpha::Dlb::FallbackPolicy
```

Comment:
ISTM that this shouldn't be a class member since it's just used when we call `createConnectionBalancerFromProto` which has all the information needed from the provided config of the fallback policy

Reply:
As `fallback` func needs `ENVOY_LOG`, make it as class func and make policy as class member will be simple for codes.